### PR TITLE
Add the style of author pages

### PR DESCRIPTION
**Pull request by cafedomancer**
Connects to bm-sms/daimon-news-misc#152.

### CODE REVIEW

File: app/assets/stylesheets/themes/default/_participant.scss [hunk 1]
```diff
@@ -0,0 +1,146 @@
+.participant-content {
```
Comment (tricknotes) [inline app/assets/stylesheets/themes/default/_participant.scss:1]: Can you extract top level name (that matches file name) such as the following?

``` scss
.participant {
  &-content {
    // ...
  }
}
```

File: app/assets/stylesheets/themes/default/_participant.scss [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  @include media($smaller-tablet) {
+    li {
+      font-size: $default-font-size - 2;
+    }
+  }
+}
+
+@include media($larger-tablet) {
```
Comment (tricknotes) [inline app/assets/stylesheets/themes/default/_participant.scss:56]: Can you embed media query into class name for consistency?

``` scss
.participant-content {
  @include media($larger-tablet) {
    // ...
  }
}
```

See also [`_post.scss`](https://github.com/bm-sms/daimon_news-layout/blob/c93b45154a49f79752b64147a668151f877a79f5/app/assets/stylesheets/themes/default/_post.scss#L11-L13).

File: app/assets/stylesheets/themes/default/_participant.scss (hunk 1 above)
Comment (cafedomancer) [inline app/assets/stylesheets/themes/default/_participant.scss:1]: The block is not `-content` but `pariticpant-content`, so I prevented from inheriting. More important to no repetition?

Comment (tricknotes) [inline app/assets/stylesheets/themes/default/_participant.scss:1]: Ah right.
I misunderstood this code.

File: app/assets/stylesheets/themes/default/_participant.scss (hunk 2 above)
Comment (cafedomancer) [inline app/assets/stylesheets/themes/default/_participant.scss:56]: Sorry, I don't know what you pointed out. Need to flip class name and media query? Which is consistent, `_post.scss` or `_serial.scss`?

Comment (tricknotes) [inline app/assets/stylesheets/themes/default/_participant.scss:56]: There is no consistency here 😇

File: app/assets/stylesheets/themes/default/_participant.scss [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+  padding: 20px;
+  margin-bottom: 20px;
+
+  &__title {
+    color: $base-color;
+    font-size: $default-font-size;
+    margin: 0;
+  }
+
+  &__description {
```
Comment (mtsmfm) [inline app/assets/stylesheets/themes/default/_participant.scss:12]: How about extracting as mixin or using extend?

``` scss
@mixin markdown-body {
  img {
    ...
  }

  ...
}

.participant-content {
  &__description {
    @include markdown-body;
  }
}

.post {
  &__body {
    @include markdown-body;
  }
}
```

or

``` scss
.participant-content {
  &__description {
    @extend .post__body
  }
}
```

Comment (cafedomancer) [inline app/assets/stylesheets/themes/default/_participant.scss:12]: I like the former because not to complicate dependencies. I will try it.

**Comment by mtsmfm** [comment]
LGTM

**Pull request merged**